### ISSUE TITLE
Broccoli build error after installing ember-x-editable and emberx-select.

### ISSUE THREAD

**Issue by bdiz** [shipshapecode/ember-x-editable#91]
I am using emberx-select in my app and seem to be getting an error when I install ember-x-editable and then build.

```
$ ember build
Could not start watchman
Visit https://ember-cli.com/user-guide/#watchman for more info.
cleaning up...
Build failed.
The Broccoli Plugin: [broccoli-persistent-filter:TemplateCompiler] failed with:
Error: EEXIST: file already exists, symlink '/home/bdiz/Dropbox/dev/ember/new-app/node_modules/emberx-select/app/templates/components/x-select.js' -> '/home/bdiz/Dropbox/dev/ember/new-app/tmp/broccoli_persistent_filtertemplate_compiler-output_path-IWzIbDKy.tmp/new-app/templates/components/x-select.js'
```

I built a new app. Install of ember-x-editable in my new app is fine until I install emberx-select and then build errors.

I am using ember 2.13.2.

**Comment by rwwagner90** [comment]
Hi @bdiz sorry for the delay! The issue here is just that each addon has a component called `x-select`. I can rename the one here, and the issue should go away.

**Comment by rwwagner90** [comment]
I just released version 0.5.0, which renames the components, but it also restructures the way things are used now. I made text and select contextual components, rather than passing type, so you now use it like so:

```hbs
{{#ember-x-editable
cancelAction='cancelAction'
fontFamilyConfig=fontFamilyConfig
isFieldEditing=isFieldEditing
saveAction='saveAction'
validator=textValidator
value='TestString' as |x|}}
  {{x.text}}
{{/ember-x-editable}}
```
